images/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM debian:stable as java8-builder
+
+RUN apt update -qq &&\
+    apt install wget -y
+
+RUN wget https://downloads.mesosphere.com/java/openjdk-jre-8u212b03-hotspot-linux-x64.tar.gz -P /srv && \
+    mkdir /srv/jdk && \
+    tar -xvvf /srv/openjdk-jre-8u212b03-hotspot-linux-x64.tar.gz --strip-components=1 -C /srv/jdk
+
+FROM debian:stretch-slim
```

Comment:
same here

Reply:
updated to `debian:stretch-20190610-slim`